### PR TITLE
Network/FormData typeof object allowing null

### PR DESCRIPTION
**Pull request by moaxaca**
Submitting PR in response to https://github.com/facebook/react-native/issues/12251

**Test plan (required)**

$formData = new FormData()
$formData.append('value', null)

Included Unit Tests

### CODE REVIEW

File: Libraries/Network/__tests__/FormData-test.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+  afterEach(() => {
+    formData = null;
+  });
+
+  it('should return non blob null', function() {
+    formData.append('null', null);
+
+    let parts = formData.getParts();
+    expect(parts[0].string).toBe('null');
```
Comment (ide) [inline Libraries/Network/__tests__/FormData-test.js]: Can you use toMatchObject here and below?

https://facebook.github.io/jest/docs/expect.html#tomatchobjectobject

**Comment by ide** [comment]
@facebook-github-bot shipit

**Comment by moaxaca** [comment]
@ide I am not 100% how the facebook-github-bot works but it appears to have repeated my CLA signing. Is that correct? I updated the branch to see if that would re-trigger the bot.

**Comment by ide** [comment]
@facebook-github-bot shipit

**Comment by moaxaca** [comment]
@ide Something seems to be up with this PR what should I do?

**Comment by ide** [comment]
@lacker do you think you could you take a look at this stuck PR?

**Comment by lacker** [comment]
OK - let me first make sure circle is green on this and then i'll merge it in if possible. My guess is some CLA bug, there have been a few of those recently.

**Comment by moaxaca** [comment]
@lacker ty

**Pull request closed without merging**